pkgs/by-name/mo/moneydance/package.nix

```
@@ -0,0 +1,56 @@
+{ lib, stdenv, fetchurl, makeWrapper, dpkg, openjdk21, openjfx21, }:
+let
+  version = "2023.3_5064";
+  hash = "sha256-t1jBZJ2sL0YDx9915UWgpCzKnpNq1Qz5i7ygGXDI5dk=";
+in stdenv.mkDerivation rec {
+  pname = "moneydance";
+  inherit version;
+
+  src = fetchurl {
+    url = "https://infinitekind.com/stabledl/${version}/moneydance_linux_amd64.deb";
+    inherit hash;
+  };
```

Useless `let` block:

And let's use the `finalAttrs` idiom:
```suggestion
stdenv.mkDerivation (finalAttrs: {
  pname = "moneydance";
  version = "2023.3_5064";

  src = fetchurl {
    url = "https://infinitekind.com/stabledl/${finalAttrs.version}/moneydance_linux_amd64.deb";
    hash = "sha256-t1jBZJ2sL0YDx9915UWgpCzKnpNq1Qz5i7ygGXDI5dk=";
  };
```

---

(Don't forget to close the parenthesis on the other side)

---

Huh, today I learned a thing!